TextInputKit/TextInputKit/Code/SpecializedTextInput/CardNumber/BankCardBrand.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BankCardBrand.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 30/11/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+
+enum BankCardBrand {
```

Comment:
It's a good tone either inherit it from Int either provide equal method (ideally, you can make it Hashable)